www/_posts/2016-02-29-docs-improvement.md

```diff
@@ -0,0 +1,39 @@
+--- 
+layout: post 
+author: 
+    name: Raghav Katyal
+    url: https://github.com/rakatyal
+title:  "Significant Documentation Updates"
+categories: announcements 
+tags: news releases docs 
+---
+
+We are happy to announce that we have made significant content updates to our documentation.
+<!--more-->
+
+A while back we conducted a [survey](https://apachecordovabot.typeform.com/to/BCc5co) on the Cordova website asking for feedback on the experience of using Cordova from the developer standpoint. We received more than 900 responses and documentation emerged as one of the major pain points for the average Cordova developer.
+
+Since then we have made efforts to improve the quality of our documentation. To summarize it:
+
+- Validated and corrected most of the current doc content.
+- Re-organized the content for our website and made our table of contents more concise and composed of action-driven items for ease of navigation.
```

Comment:
Reads like a run-on sentence. Also no hyphen in reorganized. Maybe:

Reorganized our documentation and updated our table of contents to use concise, action-driven titles for ease of navigation.